### DISCUSSION TITLE
cypress questions

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#13800] [Quasar App Extensions] [answered]
I recently installed cypress `quasar ext add @quasar/testing-e2e-cypress` and have been trying it out and have a few questions:
- if I run cypress standalone (via `npx cypress open`) it tells me there is an upgrade to version 10. I assume this is not supported within quasar?
- quasar created some component tests inside my `src/components/__tests__/`, and added several components to `src/components`, and (by examining the scripts added to package.json) I have tried running `npm run test:component` for example, but all components fail with `Module not found: Error: Can't resolve 'src/css/app.scss'` which is baffling to me when my quasar.conf.js file specifies `app.css`. Any ideas?

**Comment by yusufkandemir** [comment, marked as answer]
I want to start by letting you know that https://github.com/quasarframework/quasar-testing is a better place for this kind of questions.

1. Cypress 10 support is WIP: https://github.com/quasarframework/quasar-testing/pull/259
2. See `test/cypress/support/component.[js|ts]`:
	  ```js
	  // Change this if you have a different entrypoint for the main scss.
	  import 'src/css/app.scss';
	  ```
	So, you would need to change that to `import 'src/css/app.css';` in your case.

**Reply by ssuess** [reply]
Thanks I will pose these types of questions there in the future.
